server/controllers/ClientDashboardController.js

```diff
@@ -27,4 +27,15 @@ async function getUsername(req, res) {
   res.json({ username: clientUsername });
 }
 
-module.exports = { getActiveGigs, postGig, getUsername };
+async function getPerformerDetails(req, res) {
+  const user = req.session.userPerformer;
+  const performerInfo = await UserPerformer.find({

```

Comment:
use findById(user._id), there is a chance that the user info changed and we can never guarantee that the user data is the same in the session as it is in the database when searching for it. We do know however that 1 thing apart of the user won't change and its the _id